app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
+            raise NotVaccinatedError("not vaccinated")
+        elif vstr["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("outdated")
+        elif not vstr["wearing_a_mask"]:
+            raise NotWearingMaskError("not wearing mask")
+        else:
+            return "Welcome to " + self.name
```

Comment:
```suggestion
        if not vstr.get("vaccine"):
            raise NotVaccinatedError("not vaccinated")

        if vstr["vaccine"]["expiration_date"] < datetime.date.today():
            raise OutdatedVaccineError("outdated")

        if not vstr["wearing_a_mask"]:
            raise NotWearingMaskError("not wearing mask")

        return "Welcome to " + self.name
```
Now code looks more readable. And if we read the code it looks like the one from requirements, each statement is separated.